docs/ruff.toml

```diff
@@ -0,0 +1 @@
+line-length=78
```

Comment:
From experimentation, 78 chars is when the horizontal scroll appears on the website.